### DISCUSSION TITLE
How to use multiple filters to get ids of detects

### DISCUSSION THREAD

**Discussion by kumar4755** [CrowdStrike/psfalcon#99] [Q&A] [answered]
How to use multiple filters while getting ids of detects

Get-CsDetectId -Filter "first_behavior:<'2021-02-06'" AND "sort=first_behavior.desc"

And while using -All im getting only 10000 results is there a way to get all the detection ids

**Comment by bk-cs** [comment, marked as answer]
Sort is not a property of FQL, it is it's own property. Note that my example is for PSFalcon v2 and you appear to be using v1 (unsupported).
```
Get-FalconDetection -Filter "first_behavior:<'2021-02-06'" -Sort 'first_behavior.desc'
```
If you were trying to combine two proper FQL filters, you would use `+`:
```
Get-FalconDetection -Filter "first_behavior:<'2021-02-06'+status:'new'"
```
See the following wiki article for more information about FQL filters: https://github.com/CrowdStrike/psfalcon/wiki/Filtering-and-the-Falcon-Query-Language
